cmd/namespace/namespace.go

```diff
@@ -21,16 +21,36 @@ import (
 	"github.com/okteto/okteto/pkg/analytics"
 	"github.com/okteto/okteto/pkg/errors"
 	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/types"
 	"github.com/spf13/cobra"
 )
 
+//NamespaceCommand has all the namespaces subcommands
+type NamespaceCommand struct {
+	ctxCmd   *contextCMD.ContextCommand
+	okClient types.OktetoInterface
+}
+
+//NewNamespaceCommand creates a namespace command to
+func NewNamespaceCommand() (*NamespaceCommand, error) {
+	c, err := okteto.NewOktetoClient()
+	if err != nil {
+		return nil, err
+	}
+	return &NamespaceCommand{
+		ctxCmd:   contextCMD.NewContextCommand(),
+		okClient: c,
+	}, nil
+}
+
 // Namespace fetch credentials for a cluster namespace
 func Namespace(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:    "namespace [name]",
-		Hidden: true,
-		Short:  "Download k8s credentials for a namespace",
-		Args:   utils.MaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#namespace"),
+		Use:     "namespace [name]",
+		Hidden:  true,
+		Short:   "Download k8s credentials for a namespace",
```

Comment:
It's just configure the current namespace of the okteto context, it doesn't download k8s credentials